src/ixdat/measurements.py

```diff
@@ -439,6 +440,26 @@ def c_ids(self):
     def add_calibration(self, calibration):
         self._calibration_list = [calibration] + self._calibration_list
 
+    @property
+    @deprecate(
```

Comment:
Alright so, I was sort of expecting a problem around writing the keyword arguments out in full or not when I wrote it. I think, that since the is breaching multiple lines any, that I would as a courtesy to other developers, because the deprecate calls are not exactly self explanatory. Long story short, I think that for the ones that breach multiple lines anyway, I would use the keywords to make it easier to read.

This also helps a little with making sure that the message, which may be multi-line, is easier to separate from the next arguments. Consider how small the difference is between these two:
```python
@deprecate(
    "0.1",
    "At present, ixdat measurements have a `calibration_list` but no compound "
    "`calibration`, and the property just returns the first from the list.",
    None,
)

@deprecate(
    "0.1",
    "At present, ixdat measurements have a `calibration_list` but no compound ",
    "`calibration`, and the property just returns the first from the list.",
    None,
)
```
where the latter is a bug that fills in a nonsensical `hard_deprecation_release`. Whereas if we do:
```python
@deprecate(
    last_supported_version="0.1",
    update_message=(
        "At present, ixdat measurements have a `calibration_list` but no compound "
        "`calibration`, and the property just returns the first from the list."
    )
    hard_deprecation_release=None,
)
```
that risk is seriously mitigated.

I will not comment on this more than once.